roles/etcd/tasks/upgrade_image.yml

```diff
@@ -1,6 +1,2 @@
 ---
-- include_tasks: upgrade/upgrade_static.yml
-  when: openshift_master_bootstrap_enabled | default(False) | bool
-
 - include_tasks: upgrade/upgrade_image.yml
```

Comment:
Still need this for atomic until we determine what we want to do for atomic etcd standalone hosts.